tools/static-assets/scaffolds-ts/methods.ts

```diff
@@ -0,0 +1,25 @@
+import { Meteor } from 'meteor/meteor';
+import { $$UpperName$$Type, $$UpperName$$Collection } from './collection';
+
+export function create$$UpperName$$(data: $$UpperName$$Type) {
+  return $$UpperName$$Collection.insertAsync({ ...data });
+}
+
+export function update$$UpperName$$(_id: string, data: Partial<$$UpperName$$Type>) {
+  return $$UpperName$$Collection.updateAsync(_id, { ...data });
+}
```

Comment:
Actually, `data` should use `UpdateFilter` and not `Partial`, as right now it does a full document replace as well as doesn't support operators.

Reply:
Do you mean ``Mongo.Modifier<$$UpperName$$>``?